### ISSUE TITLE
Clarify model_name_or_path: ../llama3-chat reference

### ISSUE THREAD

**Issue by nguyenhoangchu** [Deriq-Qian-Dong/DecoupledRAG#2]
Thank you for your work. I’m trying to reproduce your results, but when I ran:

> sh scripts/rag_llama_run.sh

I noticed that you set:
 ` model_name_or_path: ../llama3-chat`

By “llama3-chat,” do you mean the `meta-llama/Meta-Llama-3-8B-Instruct` model?

Below is the full content of your `rag_llama_config.yaml`:

> trainer: RAGLanguageModelTrainer
training:
  model_type: llama
  project_name: RAG_llama3-chat-ca-rag
  model_name_or_path: ../llama3-chat
  tokenizer_name_or_path: ../llama3-chat
  project_dir: output_ca_test
  kg_model_name_or_path: RAG-best
  num_layers_unfrozen: -1
  gradient_checkpointing: false
  optimizer: 
    name: Lamb
    kwargs:
      lr: 1e-3
      weight_decay: 0.0
      betas: [0.9, 0.999]
      eps: 1e-08
  scheduler:
    name: LinearLR
    kwargs:
      total_iters: 10
  num_epochs: 100000
  log_with: tensorboard
  negatives_in_device: true
  negatives_x_device: true
  predict_from_last: 10000
  eval_step: 500
  start_from: 0
  skip_steps: 0
  cross_attention_activation_function: silu
  add_cross_attention_layer_number: 31
  knowledge_max_seq_len: 64
  kb_path: ../data_of_ReGPT/En-Wiki/phrases_embeddings.npy
  freeze_retrieval_head: true
  freeze_lm_head: true
  metrics:
    - f1
    - accuracy
  target_metric: accuracy

dataset:
  number_of_docs: 1
  inference_with_explict_docs_for_test: false
  train:
    2WikiMultihopQA:
      data_name_or_path: ../data_of_ReGPT/QA_datasets_wTop50/2WikiMultihopQA/sorted_datasets_train/
      dataset_name: QADataset4Chat
      batch_size: 1
      num_epochs: 805
      max_seq_len: 64
      knowledge_max_seq_len: 256
      dynamic_sampler: false
      corpus: ../data_of_ReGPT/Wiki-corpus/train
      system_prompt: "You are a helpful assistant. Please answer user question to the best of your ability. The answer MUST in ONE OR FEW WORDS. The answer is within the question."
      max_tokens: 4096
  test:
    2WikiMultihopQA:
      dataset_name: QADataset4ChatTest
      batch_size: 1
      data_name_or_path:  ../data_of_ReGPT/QA_datasets_wTop50/2WikiMultihopQA/sorted_datasets_dev/
      max_seq_len: 64
      knowledge_max_seq_len: 256
      max_new_tokens: 32
      index_path: ../data_of_ReGPT/Wiki-corpus/bm25_index/
      corpus: ../data_of_ReGPT/Wiki-corpus/train/
      system_prompt: "You are a helpful assistant. Please answer user question to the best of your ability. The answer MUST in ONE OR FEW WORDS. The answer is within the question."
      dynamic_sampler: false

generation_kwargs:
  min_length: 0
  max_length: 512
  do_sample: true
  top_k: 5
  top_p: 0.95

RAG_kwargs:
  faiss:
    dimension: 768
  retrieval_step: 10
  topk: 6

**Comment by CSQianDong** [comment]
Thanks for your interest. 
Yes. The llama3-chat is the local path of meta-llama/Meta-Llama-3-8B-Instruct. You can directly download and the put the HF model parameters into ../llama3-chat to reproduce the results.

**Comment by nguyenhoangchu** [comment]
Thanks for the quick response! I’ll go ahead and close this issue
